content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -2332,6 +2332,111 @@ Starting in v1.16, this annotation was removed in favor of
 [Pod Priority](/docs/concepts/scheduling-eviction/pod-priority-preemption/).
 {{< /note >}}
 
+### jobset.sigs.k8s.io/jobset-name
+
+Type: Label, Annotation
+
+Example:  `jobset.sigs.k8s.io/jobset-name: "my-jobset"`
+
+Used on: Jobs, Pods
+
+This label/annotation is used to store the name of the JobSet that a Job or Pod belongs to.
+[JobSet](https://jobset.sigs.k8s.io) is an extension API that you can deploy into your Kubernetes cluster.
```

Comment:
It's definitely used as a label _and_ as an annotation?

Reply:
Yes, most of these are set as both a label and annotation (see [here](https://github.com/kubernetes-sigs/jobset/blob/b92cbdcbe77f168a82cf12c9f349bdd053d2ae5c/pkg/controllers/jobset_controller.go#L722)).

Reply:
Odd, but let's document actual behavior.